.infrastructure/07_ecs.tf

```diff
@@ -1,3 +1,54 @@
 resource "aws_ecs_cluster" "cms_ecs_cluster" {
   name = "cms-ecs-cluster"
 }
+
+data "template_file" "cms_app" {
+  template = file("./task-definitions/cms_app.json.tpl")
+
+  vars = {
+    image                = aws_ecr_repository.strapi_image_repository.repository_url
+    fargate_cpu          = var.cms_app_cpu
+    fargate_memory       = var.cms_app_memory
+    aws_region           = var.aws_region
+    db_host              = aws_rds_cluster.cms_database_cluster.endpoint
+    db_user              = aws_rds_cluster.cms_database_cluster.master_username
+    db_password_arn      = aws_ssm_parameter.cms_database_password.arn
+    bucket_name          = aws_s3_bucket.cms_medialibrary_bucket.bucket
+    admin_jwt_secret_arn = aws_ssm_parameter.cms_admin_jwt_secret.arn
+    db_name              = aws_rds_cluster.cms_database_cluster.database_name
+    db_client            = "postgres"
+  }
+}
+
+resource "aws_ecs_task_definition" "cms_task_def" {
+  family                   = "cms-task-def"
+  execution_role_arn       = aws_iam_role.ecs_task_execution_role.arn
+  task_role_arn            = aws_iam_role.task_role.arn
+  network_mode             = "awsvpc"
+  requires_compatibilities = ["FARGATE"]
+  cpu                      = var.cms_app_cpu
+  memory                   = var.cms_app_memory
+  container_definitions    = data.template_file.cms_app.rendered
+}
+
+resource "aws_ecs_service" "cms_ecs_service" {
+  name                              = "cms-ecs"
+  cluster                           = "cms-ecs-cluster"
```

Comment:
Can we use some [reference](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/docdb_cluster#attribute-reference) instead duplicate the cluster name?

Reply:
right, make the change with the next commit